packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -1711,6 +1712,40 @@ QUnit.module('options changed', moduleSetup, () => {
     });
 });
 
+QUnit.module('focusOnDeletion', () => {
+    const items = ['item 1', 'item 2', 'item 3'];
+    QUnit.module('focusStateEnabled', function(assert) {
```

Comment:
I don't see why we need one more module. You can just move your tests up.

Reply:
[b8bba63](https://github.com/DevExpress/DevExtreme/pull/27230/commits/b8bba636980d2478c625678291fb31d98286ed70)